ceph/rados/core_workflows.py

```diff
@@ -2740,3 +2740,52 @@ def get_host_object(self, hostname):
 
         log.error(f"Could not find host object for host {hostname}")
         return None
+
+    def set_autoscale_flag(self, retry: int = 15) -> bool:
```

Comment:
Can we please consider changing the function name to `set_noautoscale_flag`?

Reply:
Any reason as to why we are going with retries instead of direct timeouts?

Reply:
+1

Reply:
just time does not give a correct view on how many times a command was executed to deem something does not work. Count gives much better idea. 

Nothing too technical about the choice of retries vs just timeout.